src/redux/categories/categoriesSlice.jsx

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categoies: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+
+  reducers: {
+    checkStatus: (state) => ({
+      ...state,
+      status: 'Under construction',
+    }),
+  },
```

Comment:
- Optional
Since the `status` property is already present in the initial state object, and since the reducer is not modifying any other properties of the state, you can just return `state` from the `checkStatus` reducer instead of returning a new state object with the same `status` property value.
This implementation would have the same effect as your implementation, but it would be more efficient because it avoids creating a new object unnecessarily.